example/ios/ApplovinMaxExample/AppDelegate.mm

```diff
@@ -0,0 +1,31 @@
+#import "AppDelegate.h"
+
+#import <React/RCTBundleURLProvider.h>
+
+@implementation AppDelegate
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
+{
+  self.moduleName = @"ApplovinMaxExample";
```

Comment:
Any occurrences of the brand should have the `L` capitalized - can you go through all occurrences and check? (e.g. project name was `AppLovinMAXExample`).

Reply:
The name is auto-generated by the template generator.   Will go through and update to `AppLovinMAX`.